pyRMSD/test/testSymmTools.py

```diff
@@ -37,9 +39,13 @@ def test_validation(self):
         except ValueError:
             self.fail("Value exception has been raised.")
     
-        symm_group_3 = [ [2,1,3] ] # A symm group must have elements of len 2
-         
+        # Old validation check fails
         self.assertRaises(ValueError, symm_groups_validation, [symm_group_3])
+        # New validation check passes
+        try:
+            symm_groups_validation_new([symm_group_3])
+        except ValueError:
```

Comment:
This is unnecessary. If the function raises `ValueError`, it will fail the test already.

Reply:
Removed the try-except block